coupons.go

```diff
@@ -32,4 +32,5 @@ type Coupon struct {
 	UniqueCodeTemplate       string      `xml:"unique_code_template,omitempty"`
 	UniqueCouponCodeCount    NullInt     `xml:"unique_coupon_codes_count,omitempty"`
 	PlanCodes                []string    `xml:"plan_codes>plan_code,omitempty"`
+	SubscriptionUUID         string      `xml:"subscription_uuid,omitempty"`
```

Comment:
Coupons can be redeemed to a subscription in the redemptions service, but I don't think a coupon has a `subscription_uuid` field that it holds. I'm guessing this was an oversight while adding the subscription uuid to the redemptions service. If I'm missing something, could you point me to the Recurly docs where you see this? Otherwise let's remove.